### PR TITLE
feat(idle): support comments on staling or closing issue/pr

### PR DESCRIPTION
**Pull request by yin1999**
Fixes: #56

If those message leave as empty, the comment will not be added.

See: https://github.com/actions/stale#stale-issue-message

### CODE REVIEW

**Comment by bsmth** [comment]
There's another one open in https://github.com/mdn/workflows/pull/123 - should we wait for that to land first?

**Comment by yin1999** [comment]
> There's another one open in #123 - should we wait for that to land first?

Sure, let's wait until #123 is landed :)

**Review by uiolee — APPROVED** [review]

File: .github/workflows/idle.yml [hunk 1]
```diff
@@ -22,6 +22,22 @@ on:
         default: 30
         required: false
         type: number
+      close-issue-meesage:
```
Comment (caugner) [inline .github/workflows/idle.yml:25]:
```suggestion
      close-issue-message:
```

File: .github/workflows/idle.yml [hunk 2]
```diff
@@ -34,6 +50,10 @@ jobs:
           days-before-close: ${{ inputs.days-before-close }}
           stale-issue-label: ${{ inputs.label }}
           stale-pr-label: ${{ inputs.label }}
+          close-issue-meesage: ${{ inputs.close-issue-meesage }}
```
Comment (caugner) [inline .github/workflows/idle.yml:53]:
```suggestion
          close-issue-message: ${{ inputs.close-issue-message }}
```

File: .github/workflows/idle.yml [hunk 3]
```diff
@@ -22,6 +22,22 @@ on:
         default: 30
         required: false
         type: number
+      close-issue-meesage:
+        description: The comment message will be added to the issues when closing it automatically.
```
Comment (caugner) [inline .github/workflows/idle.yml:26]:
```suggestion
        description: Comment to add when a stale issue is closed.
```

File: .github/workflows/idle.yml [hunk 4]
```diff
@@ -22,6 +22,22 @@ on:
...
+        required: false
+        type: string
+      stale-issue-message:
+        description: The comment message will be added to the issue when staling it automatically.
```
Comment (caugner) [inline .github/workflows/idle.yml:30]:
```suggestion
        description: Comment to add when an issue becomes stale.
```

File: .github/workflows/idle.yml [hunk 5]
```diff
@@ -22,6 +22,22 @@ on:
...
+        required: false
+        type: string
+      close-pr-message:
+        description: The comment message will be added to the PR when closing it automatically.
```
Comment (caugner) [inline .github/workflows/idle.yml:34]:
```suggestion
        description: Comment to add when a stale PR is closed.
```

File: .github/workflows/idle.yml [hunk 6]
```diff
@@ -22,6 +22,22 @@ on:
...
+        required: false
+        type: string
+      stale-pr-message:
+        description: The comment message will be added to the PR when staling it automatically.
```
Comment (caugner) [inline .github/workflows/idle.yml:38]:
```suggestion
        description: Comment to add when a PR becomes stale.
```

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, but found a typo and suggested some less verbose descriptions.

**Review by caugner — APPROVED** [review]

**Pull request merged**